### PR TITLE
add all exports used by react-apollo to entry point

### PR DESCRIPTION
**Pull request by jbaxleyiii**
TODO:

- [x] If this PR is a new feature, reference an issue where a consensus about the design was reached (not necessary for small changes) [issue](https://github.com/apollostack/react-apollo/issues/172)
- [ ] Make sure all of the significant new logic is covered by tests (@stubailo do we have a place to just test that exports exist?)
- [x] Rebase your changes on master so that they can be merged easily
- [ ] Make sure all tests and linter rules pass (see above)
- [x] Update CHANGELOG.md with your change
- [x] Add your name and email to the AUTHORS file (optional)

### CODE REVIEW

**Comment by stubailo** [comment]
There are tests for stuff like that in client.ts, but it's probably not necessary.

File: src/index.ts [hunk 1]
```diff
@@ -95,6 +101,16 @@ export {
   writeQueryToStore,
   writeFragmentToStore,
   print as printAST,
+  createFragmentMap,
```
Comment (stubailo) [inline src/index.ts]: Why do we need createFragmentMap?

**Comment by jbaxleyiii** [comment]
@stubailo its currently used https://github.com/apollostack/react-apollo/blob/master/src/graphql.tsx#L353 to read from the store via the fragment map

**Comment by stubailo** [comment]
What's the underlying issue there? That Apollo client has no synchronous way to read from the store? Could be really easy to add.

**Comment by jbaxleyiii** [comment]
@stubailo yep! If you have an idea of what that could look like, I'd be happy to add it!

**Pull request merged**